simtools/model/camera.py

```diff
@@ -410,40 +414,40 @@ def _findAdjacentNeighbourPixels(self, xPos, yPos, radius, rowColoumnDist):
 
         # First find the neighbours with the usual method and the original radius
         # which does not allow for diagonal neighbours.
-        neighbours = self._findNeighbours(xPos, yPos, radius)
+        neighbours = self._find_neighbours(x_pos, y_pos, radius)
         for i_pix, nn in enumerate(neighbours):
             # Find pixels defined as edge pixels now
             if len(nn) < 4:
                 # Go over all other pixels and search for ones which are adjacent
                 # but further than sqrt(2) away
-                for j_pix in range(len(xPos)):
+                for j_pix, _ in enumerate(x_pos):
```

Comment:
Yes, this change was made because Codacy complained. This change doesn't make a lot of sense actually cause this code is 10% slower, but that's what they recommend, so...